dashboard/lib/services/script_seed.rb

```diff
@@ -129,6 +129,8 @@ def self.seed_from_json(json_string)
       # as just another set of objects to import allows us to handle it with the same pattern as the other models.
       seed_context.levels = seed_context.script_levels.map(&:levels).flatten
       import_levels_script_levels(levels_script_levels_data, seed_context)
+
+      CourseOffering.add_course_offering(seed_context.script)
```

Comment:
Although this looks fairly straight forward, I'm sure it would have taken me a lot longer to figure out it was this easy! :-)